plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -32,12 +32,10 @@
 DBT_VERSION_STRING = DBT_VERSION.to_version_string()
 DBT_VERSION_TUPLE = (int(DBT_VERSION.major), int(DBT_VERSION.minor))
 
-if DBT_VERSION_TUPLE >= (1, 3):
-    COMPILED_SQL_ATTRIBUTE = "compiled_code"
-    RAW_SQL_ATTRIBUTE = "raw_code"
-else:
-    COMPILED_SQL_ATTRIBUTE = "compiled_sql"
-    RAW_SQL_ATTRIBUTE = "raw_sql"
+COMPILED_SQL_ATTRIBUTE = (
+    "compiled_code" if DBT_VERSION_TUPLE >= (1, 3) else "compiled_sql"
+)
+RAW_SQL_ATTRIBUTE = "raw_code" if DBT_VERSION_TUPLE >= (1, 3) else "raw_sql"
```

Comment:
The `if/else` was causing uncovered lines. Rather than adding `pragma: no cover`, I changed to use the ternary operator.